app/src/pages/inside/common/utils.js

```diff
@@ -17,8 +17,83 @@
 import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
 import { canBulkEditItems } from 'common/utils/permissions';
 import { isPostIssueActionAvailable } from 'controllers/plugins';
+import {
+  AUTOMATION_BUG,
+  DEFECT_TYPES_SEQUENCE,
+  NO_DEFECT,
+  PRODUCT_BUG,
+  SYSTEM_ISSUE,
+  TO_INVESTIGATE,
+} from 'common/constants/defectTypes';
+import { defineMessages } from 'react-intl';
 import { actionMessages, ISSUE_OPERATION_MAX_ITEMS } from './constants';
 
+const DEFECT_STATISTICS_BASE = 'statistics$defects$';
+
+const messages = defineMessages({
+  [PRODUCT_BUG]: {
```

Comment:
Why are widget's messages in the utils? Move them to a more suitable place, please.